week-1/Homework/mandatory/6-playing-computer.js

```diff
@@ -8,11 +8,17 @@
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
   2. Remove the line that throws the error.
+  // The line 31 throws the error because we don't have a defined variable b.
   3. What is printed to the console?
+  // The console prints first the variabels x and a, then the numbers: 4,9,6,13,8.
   4. How many times is "f1" called?
+  //Three times
```

Comment:
Questions 1-3 are right here, but 4-7 are slightly off - can you give them another go? It may help to write out the values of all of the variables for each iteration, i.e.
```
i=0: x=2, a=3, f2(a=0, b=2), d=4
i=1: ...
...
```